spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowControllerAutoConfiguration.java

```diff
@@ -34,6 +34,7 @@
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.boot.web.client.RestTemplateBuilder;
 import org.springframework.cloud.common.security.AuthorizationProperties;
+import org.springframework.cloud.common.security.support.OAuth2AccessTokenProvidingClientHttpRequestInterceptor;
```

Comment:
Unused import.

Reply:
And Line 375 needs to be removed.

Reply:
This is being used for communicating with Skipper.

Reply:
At this point its failing to find the import and failing the compile.